index.html

```diff
@@ -1810,11 +1810,13 @@ <h3>DID Subject</h3>
         </pre>
 
         <p class="note" title="Intermediate representations">
-[=DID method=] specifications can create intermediate representations of a
-[=DID document=] that do not contain the `id` property,
-such as when a [=DID resolver=] is performing [=DID resolution=].
-However, the fully resolved [=DID document=] always contains a valid
-`id` property.
+[=DID method=] specifications can create intermediate representations of a [=DID
+document=], such as when a [=decentralized identifier=] is being registered in a
+[=verifiable data registry=] or when a [=DID resolver=] is performing [=DID
+resolution=]. These intermediate representations might not contain `id` values,
+or might contain interim values for certain `id` properties. Once the [=DID
+document=]-related operation is complete, the final `id` value will be
+determined and put in place.
```

Comment:
```suggestion
determined and put in place of the interim values throughout the [=DID document=].
```

I want to get at that, once the `id` value is calculated it is not just the `id` properties that might need updating. E.g. `verificationRelationships` might reference a vm that used the interim DID value as part of its identifier. Also the controller property may have used the interim value.